overrides/groovy/postInit/Post-Initial/Main/General/Misc/projectred.groovy

```diff
@@ -0,0 +1,77 @@
+import com.nomiceu.nomilabs.util.LabsModeHelper
+
+import static com.nomiceu.nomilabs.groovy.GroovyHelpers.RecipeRecyclingHelpers.*
+import static gregtech.api.GTValues.*
+
+//Conductive Plate
+mods.gregtech.assembler.recipeBuilder()
+        .inputs(metaitem('plateRedAlloy'), item('projectred-core:resource_item'))
+        .circuitMeta(7)
+        .outputs(item('projectred-core:resource_item', 1))
+        .duration(100).EUt(16)

```

Comment:
Please use GT Values Constants Here, and through the rest of the recipes as well.

Let's use `VA[LV]` (an EUt of 30).

Reply:
done, changed all EUt values from 16 to VA[LV]